composer.json

```diff
@@ -5,7 +5,14 @@
     "description": "This package provides wrapper functionality for the Elasticsearch engine.",
     "require": {
         "ext-curl": "*",
-        "neos/flow": "~4.0 | dev-master"
+
+        "doctrine/annotations": "^1.0",
+        "doctrine/collections": "^1.0",
+        "doctrine/orm": "^2.0",
+        "neos/error-messages": "^4.0",
+        "neos/flow": "^4.0",
+        "neos/utility-arrays": "^4.0",
+        "neos/utility-objecthandling": "^4.0"
```

Comment:
Why list transitive dependencies of `neos/flow` again? As a safeguard against changes? Not needed, since changes to that IMHO cannot happen unless a major version of Flow itself happens.